README.md

```diff
@@ -200,19 +200,18 @@ settings.to_dict()
 # }
 ```
 
-## Валидация
-При переопределении метода `update_settings_from_source` рекомендуется использовать следующие методы класса `BaseSettings`:
-
-1. `pre_validate` — проверка наличия типов в `Property`, проверка соответствия типу `Property` в возможных значениях `Property.choices`, проверка значения по умолчанию на соответствии типу.
-
-2. `post_validate` — проверка на то, что каждому `Property` присвоено какое-либо значение
+## Validation
+It is recommended to use following `BaseSettings` class methods during redefinition `update_settings_from_source` method:
+    
+1. `pre_validate` - check for the presence of types in` Property`, check for compliance with the type `Property` in possible values ​​of` Property.choices`, check for default values ​​for type matching.
+2. `post_validate` - check if each `Property` is assigned a value.
   
 
-Динамические настройки
-----------------------
+Dynamic settings
+----------------
 
-### Объявление класса, реализующего работу с источником настроек
-На примере хранения настроек в словаре `source`:
+### Definition of class working with settings source:
```

Comment:
MD022/blanks-around-headings/blanks-around-headers: Headings should be surrounded by blank lines 
MD026/no-trailing-punctuation: Trailing punctuation in heading

```suggestion
### Definition of class working with settings source

```